web-marketplace/src/pages/BuyCredits/BuyCredits.utils.ts

```diff
@@ -0,0 +1,44 @@
+import { msg } from '@lingui/macro';
+
+import { Project } from 'generated/sanity-graphql';
+import { TranslatorType } from 'lib/i18n/i18n.types';
+
+import { UISellOrderInfo } from 'pages/Projects/AllProjects/AllProjects.types';
+
+import { PAYMENT_OPTIONS } from './BuyCredits.constants';
+import { PaymentOptionsType } from './BuyCredits.types';
+
+type GetFormModelParams = {
+  _: TranslatorType;
+  paymentOption: PaymentOptionsType;
+  retiring: boolean;
+  projectId: string;
+};
+export const getFormModel = ({
+  _,
```

Comment:
Another way that I think would be more efficient would be that instead of passing around `_` we can use `i18n` from `import { i18n } from '@lingui/core';`

That way we could directly do: ``i18n._(msg`Choose credits`)``

Reply:
Reading https://lingui.dev/ref/react#uselingui, I believe this might cause some issues when the user switches languages

Reply:
You're right that there could be issues when switching languages. However, what I understand is that it can cause issues when `i18n._()` is used in memoized functions or components. As long as we avoid using it in those contexts it seems that it should work fine.